src/router/index.js

```diff
@@ -2,11 +2,11 @@
 import Vue from 'vue';
 import Router from 'vue-router';
 
-import ExampleOne from '@/pages/ExampleOne';
 import ExampleTwo from '@/pages/ExampleTwo';
 import ExampleThree from '@/pages/ExampleThree';
 import ExampleFour from '@/pages/ExampleFour';
-const Home = () => import(/* webpackChunkname: "home" */'@/pages/Home');
+const Home = () => import(/* webpackChunkName: "Home" */'@/pages/Home');
```

Comment:
Changing `webpackChunkname` to `webpackChunkName` (notice the capital N) gets the `[name]` part working again.